sw/device/tests/BUILD

```
@@ -511,6 +512,53 @@ opentitan_functest(
     ],
 )
 
+otp_json(
+    name = "otp_json_uncalibrated_clocks",
+    partitions = [
+        otp_partition(
+            name = "CREATOR_SW_CFG",
+            items = {
+                # Enable AST initialization.
+                "CREATOR_SW_CFG_AST_INIT_EN": otp_hex(CONST.MUBI4_FALSE)
+            },
+        ),
+    ],
+)
+
+otp_image(
+    name = "otp_uncalibrated_clocks",
+    verbose = True,
+    src = "//hw/ip/otp_ctrl/data:otp_json_test_unlocked0",
+    overlays = STD_OTP_OVERLAYS + [":otp_json_uncalibrated_clocks"],
+    visibility = ["//visibility:private"],
+)
+
+opentitan_functest(
+    name = "clkmgr_uncalibrated_test",
+    srcs = ["clkmgr_uncalibrated_test.c"],
+    verilator = verilator_params(
+        tags = ["broken"],  # FIXME #16374 test doesn't make progress after enabling PWM
+    ),
```

you can get rid of this completely, and just say this test only runs in DV with:
```suggestion
    targets = ["dv"],
```